build/Linux+BSD/jessie-crosscompile-armhf.cmake

```diff
@@ -0,0 +1,52 @@
+#Cross toolchain file for building QT project (specifically MuseScore)
```

Comment:
All of your files should be in "build/Linux+BSD/portable", or you should make a new directory. Something like "build/Linux+BSD/portable-ARM" or "build/Linux+BSD/portable/ARM".